NineChronicles.Headless/GraphTypes/States/AgentStateType.cs

```diff
@@ -0,0 +1,44 @@
+using System.Linq;
+using Bencodex.Types;
+using GraphQL;
+using GraphQL.Types;
+using Libplanet.Action;
+using Libplanet.Assets;
+using Libplanet.Blockchain;
+using Nekoyume.Action;
+using Nekoyume.Model.State;
+
+namespace NineChronicles.Headless.GraphTypes.States
+{
+    public class AgentStateType : ObjectGraphType<AgentState>
+    {
+        public AgentStateType()
+        {
+            Field<NonNullGraphType<AddressType>>(
+                nameof(AgentState.address),
+                resolve: context => context.Source.address);
+            Field<ListGraphType<NonNullGraphType<AddressType>>>(
+                nameof(AgentState.avatarAddresses),
+                resolve: context => context.Source.avatarAddresses.Select(a => a.Value));
+            Field<NonNullGraphType<StringGraphType>>("gold",
+                resolve: context =>
+                {
+                    StandaloneContext standaloneContext = (StandaloneContext) context.UserContext["standAloneContext"];
```

Comment:
I'm not sure it is the best way to use `UserContext`. Because `UserContext` is typed as `IDictionary<string, object>` and it doesn't guarantee it is an instance of the expected type (i.e. `StandaloneContext`). I suggest to turn into *Dependency Injection*.

I was not sure it can be proceeded without a problem, so I tried this idea at https://github.com/moreal/NineChronicles.Headless/commit/84aa8b592b417882d879f34e3b19b135d5db54f8. It worked in my local environment. You can reference it to see how it changed and to apply it if you accepted.

You can reference also [the graphql-dotnet *Dependency Injection* document](https://graphql-dotnet.github.io/docs/getting-started/dependency-injection).